src/x86_64/mp.c

```diff
@@ -48,11 +48,11 @@ void init_cpu_features()
     cr &= ~C0_EM;
     mov_to_cr("cr0", cr);
     if (use_xsave) {
-        if (v[2] & CPUID_AVX) {
-            xgetbv(0, &v[0], &v[1]);
-            v[0] |= XCR0_SSE | XCR0_AVX;
-            xsetbv(0, v[0], v[1]);
-        }
+        xgetbv(0, &v[0], &v[1]);
+        v[0] |= XCR0_SSE;
```

Comment:
Have you found something that says the presence of xsave implies the presence of sse? I think that's true, but I haven't found documentation to confirm yet.

Reply:
The Intel SDM (vol. 1, section 13.2, CPUID function 0DH, sub-function 0) says that the presence of xsave guarantees that the SSE state can be managed by xsave/xrstor. I think this implies that SSE instructions are supported, since it would make little sense to save/restore the SSE state if this state cannot be used by software. But I didn't find any explicit sentence that says that every cpu that supports xsave must also support SSE instructions.